src/resources/js/admin-log-controls.js

```diff
@@ -91,6 +91,12 @@ var tribe_logger_data  = tribe_logger_data || {};
 	 */
 	function update_download_link() {
 		var url = $download_link.attr( 'href' );
+
+		// bail if not in DOM
```

Comment:
Should there be a `$download_link` check too, in case it's not there?

Reply:
Yeah, it can be done both ways. The href is more specific, but if there's no download link it can bail too.

Reply:
My worry was mainly that .attr() would trigger a console error or something if $download_link wasn't found. Thanks!